projects/addon-mobile/components/sheet/components/sheets-host/sheets-host.template.html

```diff
@@ -1,6 +1,6 @@
-<ng-container *ngIf="service.sheets$ | async as items">
+<ng-container *ngIf="sheets.length > 0">
```

Comment:
We do not need this `ng-container` and `*ngIf`

Reply:
What about the overlay? Without the container it would always render that and it has a reference to `sheets[0]`.

Reply:
It is hidden with CSS unless there is a Sheet with overlay visible.